XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1696,6 +1691,20 @@ private string hostInstalledSuppPacks(Host host)
             return string.Join("\n", result.ToArray());
         }
 
+        private void SetupDeprecationBanner()
+        {
+            if (!(xenObject is DockerContainer) || Helpers.ContainerCapability(xenObject.Connection))
+            {
+                Banner.Visible = false;
+            }
+            else
+            {
+                Banner.BannerType = DeprecationBanner.Type.Removal;
+                Banner.WarningMessage = string.Format(Messages.CONTAINER_MANAGEMENT_REMOVAL_WARNING, string.Format(Messages.XENSERVER_8_2, BrandManager.ProductVersion82));
+                Banner.Visible = true;

```

Comment:
Slightly unrelated to the PR.

In the `DeprecationBanner`, if there is no value for the `Details` link, the link is still shown to the User as clickable. It would probably be best to hide it since clicking it doesn't do anything.

Since it's not related to the changes here I will be approving the PR. However if you have time I'd remove the visibility of the Details link when there's no specified link, like in this case. (Or even better, also link something here!)

Reply:
I made it point to the deprecated features page.